dev/reference/tools/data-migration/from-aurora.md

```diff
@@ -0,0 +1,197 @@
+---
+title: Data Migration from AWS Aurora MySQL to TiDB
+summary: Using DM to migrate from AWS Aurora MySQL to TiDB
+category: tools
+---
+
+# Data Migration from AWS Aurora MySQL to TiDB
+
+This document describes how to use DM to migrate from [AWS Aurora MySQL](https://aws.amazon.com/cn/rds/aurora/details/mysql-details/) to TiDB。
```

Comment:
we can update links to AWS Aurora from Chinese Ver to English Ver, `https://aws.amazon.com/rds/aurora/details/mysql-details/` for this one.

(if you open the link above, it may be redirected to the Chinese Ver, don't worry because it can work for English user)

Reply:
@huangxiuyan I added the link suggested by @csuzhangxc to my suggestion for this line.